whisper-fine-tuning-event/fine-tune-whisper-streaming.ipynb

```diff
@@ -357,6 +359,32 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "666b9ef0-7909-4e1e-a419-87604d233e29",
+   "metadata": {},
+   "source": [
+    "Finally, we filter any training data with audio samples longer than 30s. These samples would otherwise be truncated by the Whisper feature-extractor which could affect the stability of training:"
```

Comment:
The Whisper feature extractor pads/truncates all samples to 30s. We should filter any **training data** with audio longer than 30s. This should improve the stability of training. 

Suppose we have a training sample:

"The cat sat ... and then he got up." 

where the last part "... and then he got up." takes place **after** 30s, the audio for this sample would be truncated to "The cat sat...", but the text would still be "The cat sat ... and then he got up."

This mis-match would affect the stability of training.